src/Protocol/Version1.php

```diff
@@ -97,7 +97,7 @@ public static function sign(string $data, AsymmetricSecretKey $key, string $foot
         $rsa = self::getRsa();
         $rsa->loadKey($key->raw());
         $signature = $rsa->sign(
-            Util::preAuthEncode([$header, $data, $footer])
+            Util::preAuthEncode(...[$header, $data, $footer])

```

Comment:
Why adding `...` instead of removing the `[` and `]`? Just curious ;)

Reply:
`...` is how you'd use it in the general case (i.e. if you had an array to pass in), so I went with that in all the changes – that said, in hindsight it's probably slightly easier on the eye to just pass the arguments individually ;-) (I think @paragonie-scott changed this after merge anyway)

Reply:
I did! :)

Reply:
This is my favourite waste of characters, legitimised by the fact that the string says "empty array" ;-)

https://github.com/paragonie/paseto/blob/c44d33467662f23d3d6ef7ea22f8926333a8e763/tests/UtilTest.php#L78-L82

Reply:
Thx for the explanation! :)